metadata/metadata_service/proxy/atlas_proxy.py

```diff
@@ -378,19 +255,19 @@ def _serialize_columns(self, *, entity: AtlasEntityWithExtInfo) -> \
         else an empty list.
         """
         columns = list()
-        for column in entity.entity[CommonParams.rels].get('columns') or list():
+        for column in entity.entity[AtlasCommonParams.rels].get('columns') or list():
             column_status = column.get('entityStatus', 'inactive').lower()
 
             if column_status != 'active':
                 continue
 
-            col_entity = entity.referredEntities[column[CommonParams.guid]]
-            col_attrs = col_entity[CommonParams.attrs]
+            col_entity = entity.referredEntities[column[AtlasCommonParams.guid]]
+            col_attrs = col_entity[AtlasCommonParams.attrs]
             statistics = list()
 
             badges = list()
             for column_classification in col_entity.get('classifications') or list():
-                if column_classification.get('entityStatus') == Status.ACTIVE:
+                if column_classification.get('entityStatus') == AtlasStatus.ACTIVE:
```

Comment:
Suggestion: this can also be a utility method in amundsen common to check if the entity is active. This is being used in this proxy in a number of places. 
`entity.is_active()`

Reply:
I've generalized most of it by introducing _filter_active helper metod in AtlasProxy. Remaining cases of using explicit `== AtlasStatus.ACTIVE` would not be easily generalized as:
- there are only 4 comparisons `== AtlasStatus.ACTIVE` outside _filter_active
- two out of those 4 rely on entity.status property
- one of those 4 rely on enitity.get('entityStatus') value (which cannot be retrieved with `.status`)
- one of those 4 rely on entity.get('relationshipStatus') value
so there is no more room for generalization IMO